### ISSUE TITLE
Reset admin script

### ISSUE THREAD

**Issue by luis100** [keeps/roda#537]
Allow to forcibly reset the admin password using a script on the back-end. The script should
- [ ] Ask for the new password
- [ ] Ensure administrators group exist and that it has all roles
- [ ] Ensure that admin belongs to the administrators group

**Comment by rui-castro** [comment]
@luis100 Do you want me to create a new project for this tool or use one of the existing projects, like `roda-common-utils`?

**Comment by rui-castro** [comment]
@luis100 or maybe `roda-core` is the best option since it already has all dependencies to deal with Apache DS.

**Comment by luis100** [comment]
@rui-castro please be aware of how deployment happens, it has zero-configuration, one roda-wui war is needed. So if this is a side-thing (a jar for cli tools) it should be easy to deploy.

You will need roda-core anyway to be able to get the configurations. So to make things simple, you could add a main to this jar file, so it allows to do the CLI.

I recommend you to use [Apache Commons CLI](https://commons.apache.org/proper/commons-cli/) to create a main in roda-core, and define it in the roda-core manifest. We could ship a roda-core fat jar that would be able to achieve the desired functionality.

**Comment by rui-castro** [comment]
@luis100 ok

**Comment by luis100** [comment]
Something like this?

```bash
$ java -jar roda-core-with-all-dependencies.jar --user-admin-reset --group-administrators-reset
New admin password: *****
Repeat admin password: *****
Admin user reset
Administrator group reset
```

**Comment by hsilva-keep** [comment]
@luis100 @rui-castro please be aware that that jar already can be generated (RodaCoreFactory is the mainclass), via maven profile **roda-core-jar** & and makes available the execution of simple tasks (but it can be improved) & the instantiation of roda-core in worker mode (for distributed multi-worker scenario).

**Comment by rui-castro** [comment]
@luis100 @hsilva-keep RodaCoreFactory main method (https://github.com/keeps/roda/blob/master/roda-core/roda-core/src/main/java/org/roda/core/RodaCoreFactory.java#L1366) doesn't use Apache Commons CLI.

**Comment by hsilva-keep** [comment]
@rui-castro who said RodaCoreFactory uses Apache Commons CLI? I just said that a main already exists and it should be taken into consideration.

**Comment by rui-castro** [comment]
@hsilva-keep @luis100 recommended using Apache Commons CLI, but the existing main doesn't use it. That's the purpuse of my remark.

**Comment by rui-castro** [comment]
@luis100 @hsilva-keep check my PR #548 
Running from a normal terminal doesn't work because of issue #547.

**Comment by luis100** [comment]
Didn't get #547 error, but got this one:

```
roda git:(rui-castro-rcastro_537_reset_admin_pass) java -jar roda-core/roda-core/target/roda-core-2.0.0-SNAPSHOT-jar-with-dependencies.jar reset user-admin
New admin password: 
Repeat admin password: 
Exception in thread "main" java.lang.NullPointerException
	at org.roda.core.RodaCoreFactory.resetAdminAccess(RodaCoreFactory.java:1379)
	at org.roda.core.RodaCoreFactory.mainMasterTasks(RodaCoreFactory.java:1436)
	at org.roda.core.RodaCoreFactory.main(RodaCoreFactory.java:1450)
```

Plus, there is no syntax information when running without arguments or just with the "reset" one.

**Comment by rui-castro** [comment]
@luis100 this is exactly the error caused by the improper init of RodaCoreFactory. If you check the logs you'll probably will see the error I mentioned in #547.

**Comment by rui-castro** [comment]
@luis100 @hsilva-keep done in PR #548 
Usage is:
```bash
java -jar roda-core-with-all-dependencies.jar reset admin
```

Also added usage message for reset params. To see the message use this command or type a wrong param.
```bash
java -jar roda-core-with-all-dependencies.jar reset
```

**Comment by luis100** [comment]
checked and merged into 26cefe2fe2cb123392cee232caa680999d94db43
